tango_ros_common/tango_ros_common/src/main/java/eu/intermodalics/tango_ros_common/TangoServiceClientNode.java

```diff
@@ -171,7 +171,7 @@ private Boolean getMapUuids() throws ServiceNotFoundException {
         ServiceClient<GetMapUuidsRequest, GetMapUuidsResponse> saveMapService =
                 mConnectedNode.newServiceClient(
                         NodeNamespaceHelper.BuildTangoRosNodeNamespaceName(GET_MAP_UUIDS_SRV_NAME),
-                        SaveMap._TYPE);
+                        GetMapUuids._TYPE);
```

Comment:
There seems to be mismatch here between saveMapService and GetMapUuids._TYPE?  Shouldn't `saveMapService` be renamed to `getMapUuidsService`?